inflate.c

```diff
@@ -1248,7 +1213,8 @@ int32_t Z_EXPORT PREFIX(inflateCopy)(PREFIX3(stream) *dest, PREFIX3(stream) *sou
         return Z_MEM_ERROR;
     window = NULL;
     if (state->window != NULL) {
-        window = (unsigned char *)ZALLOC_WINDOW(source, 1U << state->wbits, sizeof(unsigned char));
+        wsize = 1U << state->wbits;
+        window = (unsigned char *)ZALLOC_WINDOW(state->strm, wsize + state->chunksize, 2);
```

Comment:
This allocates `(wsize + state->chunksize) * 2)` when we only need `(wsize * 2) + state->chunksize`... I know it's just few bytes, but when `chunksize` gets bigger with AVX512 and later, it will be quite a lot of extra bytes.

Reply:
I think that `deflate.c @ line 253` also suffers from the same problem.

Reply:
Fixed.